nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix

```
@@ -47,9 +47,7 @@ in {
         };
       };
 
-      services.redis = {
-        enable = true;
-      };
+      services.redis.servers."".enable = true;
```

Should we name this?

```suggestion
      services.redis.servers."nextcloud".enable = true;
```

---

done.